src/main/scala/com/gu/mobile/notifications/football/lib/FootballData.scala

```diff
@@ -17,6 +17,44 @@ class FootballData(
   syntheticEvents: SyntheticMatchEventGenerator
 ) extends Logging {
 
+  implicit class RichMatchDay(matchDay: MatchDay) {
+
+    private lazy val internationalTeamsForFriendlies = Set(
+      "497",    //England,
+      "499",    //'Scotland,
+      "630",    //Wales,
+      "494",    //Republic of Ireland,
+      "964",    //Northern Ireland,
+      "999",    //Spain,
+      "1678",   //Germany,
+      "717",    //Italy,
+      "619",    //France,
+      "997",    //Belgium,
+      "5539",   //Portugal,
+      "1661",   //Turkey,
+      "629",    //Poland,
+      "716",    //Norway,
+      "5845",   //Sweden,
+      "986",    //Denmark,
+      "5827",   //Russia,
+      "965",    //Argentina,
+      "23104"   //Brazil
+    )
+
+    def isUncoveredInternationalFriendly: Boolean = Set(matchDay.homeTeam.id, matchDay.awayTeam.name).intersect(internationalTeamsForFriendlies).isEmpty
+
+    def isEarlyQualifyingRound: Boolean = competitionRoundToInt.map( r => r < 3 ).getOrElse(true)
+
+    private def competitionRoundToInt: Option[Int] = {
+      try {
+        Some(matchDay.round.roundNumber.toInt)
```

Comment:
I think `Try(matchDay.round.roundNumber.toInt).toOption` would do what you have here